mllib/src/main/scala/org/apache/spark/ml/source/libsvm/LibSVMOptions.scala

```diff
@@ -41,11 +41,15 @@ private[libsvm] class LibSVMOptions(@transient private val parameters: CaseInsen
     case o => throw new IllegalArgumentException(s"Invalid value `$o` for parameter " +
       s"`$VECTOR_TYPE`. Expected types are `sparse` and `dense`.")
   }
+
+  val lineSeparator: Option[String] = parameters.get(LINE_SEPARATOR)
```

Comment:
Could we put this option in a single place for these formats? I feel putting this option in each format looks a little messy...